package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Web application (server + web client) for VelociRaffle",
   "main": "index.js",
   "scripts": {
+    "build": "babel server -d serverDist --presets es2015",
     "precommit": "jscs . --fix && eslint ./client/app && npm test",
+    "postinstall": "npm run build",
     "dev-client": "gulp",
-    "dev-server": "nodemon server.js",
+    "dev-server": "nodemon server/index.js --exec babel-node --presets es2015",
```

Comment:
Right, this has Nodemon execute Babel CLI and passes the presets `es2015` to it. In the next PR I put the presets in a `.babelrc` file, but this will do for now.